shipyard_blueprints/salesforce/shipyard_salesforce/cli/csv-header-assist.py

```diff
@@ -0,0 +1,146 @@
+import os
+import sys
+import csv
+import argparse
+
+from shipyard_salesforce import SalesforceClient
+from shipyard_templates import ExitCodeException
+
+
+def get_mock_args():
```

Comment:
I can remove if we don't want this out in production repo- figured it saved some keystrokes when testing locally

Reply:
Yes remove this please